src/index.ts

```
@@ -152,34 +172,45 @@ async function main() {
         const pathToRedirects = join(_yargs.argv.input, REDIRECTS_FILENAME);
         const pathToLintConfig = join(_yargs.argv.input, LINT_CONFIG_FILENAME);
 
-        await processPages(outputBundlePath);
-
-        // process additional files
-        switch (outputFormat) {
-            case 'html':
-                processAssets(outputBundlePath);
-                break;
-            case 'md': {
-                shell.cp('-r', resolve(pathToConfig), userOutputFolder);
-
-                try {
-                    shell.cp('-r', resolve(pathToRedirects), userOutputFolder);
-                } catch { }
-
-                try {
-                    shell.cp('-r', resolve(pathToLintConfig), userOutputFolder);
-                } catch {}
-
-                break;
+        /* Initialize workers in advance to avoid a timeout failure due to not receiving a message from them */
+        await initLinterWorkers();
+
+        await Promise.all([
+            processLinter(),
+            processPages(outputBundlePath),
+        ]);
+
+        if (!lintOnly) {
+            // process additional files
+            switch (outputFormat) {
+                case 'html':
+                    processAssets(outputBundlePath);
+                    break;
+                case 'md': {
+                    shell.cp('-r', resolve(pathToConfig), userOutputFolder);
+
+                    try {
+                        shell.cp('-r', resolve(pathToRedirects), userOutputFolder);
+                    } catch {
```

Are we sure to ignore the errors in catch block?

---

Refactored
Shell does not throw an error, it puts an error message in stderr